go/enclave/enclave.go

```diff
@@ -1248,9 +1248,23 @@ func (e *enclaveImpl) HealthCheck() (bool, common.SystemError) {
 		e.logger.Info("HealthCheck failed for the enclave storage", log.ErrKey, err)
 		return false, nil
 	}
+
 	// todo (#1148) - enclave healthcheck operations
```

Comment:
Is this todo now done and should be removed?

Reply:
I'd leave is as I think there might be more areas to health check in the enclave.